language/java/run-containers.md

```diff
@@ -0,0 +1,185 @@
+```
+
+You should now see several containers listed. These are containers that we started and stopped, but have not been removed.
+
+Let’s restart the container that we just stopped. Locate the name of the container we just stopped and replace the name of the container below using the `restart` command.
+
+```console
+$ docker restart trusting_beaver
+```
+
+Now, list all the containers again using the `docker ps` command.
+
+```console
+$ docker ps -a
+CONTAINER ID   IMAGE                                                      COMMAND                  CREATED          STATUS                      PORTS                    NAMES
+5ff83001608c   java-docker                                                "./mvnw spring-boot:…"   10 minutes ago   Up 2 seconds                0.0.0.0:8080->8080/tcp   trusting_beaver
+630f2872ddf5   java-docker                                                "./mvnw spring-boot:…"   16 minutes ago   Exited (1) 13 minutes ago                            modest_khayyam
+a28f9d587d95   java-docker                                                "./mvnw spring-boot:…"   22 minutes ago   Exited (1) 16 minutes ago                            lucid_greider
+```
+
+Notice that the container we just restarted has been started in detached mode and has port 8080 exposed. Also, observe the status of the container is “Up X seconds”. When you restart a container, it starts with the same flags or commands that it was originally started with.
+
+Now, let’s stop and remove all of our containers and take a look at fixing the random naming issue. Find the name of your running container and replace the name in the command below with the name of the container on your system.
+
+```console
+$ docker stop trusting_beaver
+trusting_beaver
+```
+
+Now that our container is stopped, let’s remove it. When you remove a container, the process inside the container will be stopped and the metadata for the container will been removed.
+
+To remove a container, simple run the `docker rm` command passing the container name. You can pass multiple container names to the command using a single command. Again, replace the container names in the following command with the container names from your system.
+
+```console
+$ docker rm trusting_beaver modest_khayyam lucid_greider
+trusting_beaver
+modest_khayyam
+lucid_greider
+```
+
+Run the `docker ps --all` command again to see that all containers are removed.
+
+Now, let’s address the random naming issue. The standard practice is to name your containers for the simple reason that it is easier to identify what is running in the container and what application or service it is associated with.
+
+To name a container, we just need to pass the `--name` flag to the `docker run` command.
+
+```console
+$ docker run -d -p 8080:8080 --name springboot-server java-docker
```

Comment:
@glours Should we add `--rm` here as well?
Otherwise it will break when the user should run a new container with the MySQL connection in language/java/develop.md  line 67.

Reply:
Yes sure 👍